packages/app/src/app/pages/Sandbox/Editor/Workspace/WorkspaceItem/index.tsx

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
```suggestion
import React from 'react';
```

useState is unnecessary here